modules/environment/login/login-inner.nix

```diff
@@ -16,6 +16,12 @@ writeText "login-inner" ''
     echo "If nothing works, open an issue at https://github.com/t184256/nix-on-droid/issues or try the rescue shell."
   fi
 
+  ${lib.optionalString config.supervisord.enable ''
+    (if [ ! -e "${config.supervisord.socketPath}" ]; then
```

Comment:
This check fails when the supervisord socket file still exists but supervisor isn't running. I'm not sure what a good alternative would be.

Reply:
`nc -zU ${config.supervisord.socketPath}` will fail if the socket file isn't actually connected, and succeed without actually sending anything on the socket otherwise.